types/redux-action/package.json

```diff
@@ -1,7 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "redux": "^3.6.0",
-        "redux-thunk": "^2.2.0"
```

Comment:
You might remove [this line](https://github.com/Microsoft/types-publisher/blob/master/dependenciesWhitelist.txt#L46) from `types-publisher` if `redux-thunk` isn't going to be distributing its own types any longer.

Reply:
Oh, I didn't know about that. Thanks for the heads up!